### DISCUSSION TITLE
2024/09/wang-fu-jing/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#106] [General]
# 2024/09/wang-fu-jing/

https://xiangyun.rbind.io/2024/09/wang-fu-jing/

**Comment by XiangyunHuang** [comment]
就在文章写完之后没多久，股市大水漫灌，楼市政策松绑，来的越猛烈，说明危机越严重，真正的历史拐点要来了。

**Reply by XiangyunHuang** [reply]
股市里的普通股民若不割了退出来，危机一旦炸开，将血本无归。

**Reply by XiangyunHuang** [reply]
果然，今天再去看 A 股指数，过去一周多时间一直跌，很多人觉得这波操作不是割韭菜而是拔韭菜。

**Reply by XiangyunHuang** [reply]
接下来，在一两个月的短期内，商品会出现一波通胀，然后，年后会继续更加恶劣的通缩。
